tests/cpp_api/audio_tests/audio_tests.cpp

```diff
@@ -132,12 +144,17 @@ int test(int test_case, const char *path, int qa_mode, int downmix, int gpu) {
         return -1;
     }
 
-    std::cout << "Running LABEL READER" << std::endl;
-    rocalCreateLabelReader(handle, path);
+    std::string file_list_path = "";  // User can modify this with the file list path if required
```

Comment:
std::string constructor will create empty string by default. So = "" is not required. Please remove all such cases in other instances